swagger-templates/src/main/resources/OktaJavaImpl/apiMethodPostResource.mustache

```diff
@@ -27,7 +27,12 @@
         {{/vendorExtensions.bodyIsSelf}}
         {{^vendorExtensions.bodyIsSelf}}
             {{#bodyParam}}
+                {{#bodyParam.required}}
             {{bodyParam.paramName}},
+                {{/bodyParam.required}}
```

Comment:
The indentation makes this a little difficult to read, but I'm assuming it is necessary because it'll throw off the rendered template, correct?

If not, can we make these line-up so it is easier to read?:
```
{{#bodyParam}}
  {{#bodyParam.required}}
  {{bodyParam.paramName}},
  {{/bodyParam.required}}
...
```

Reply:
Correct, it has been bugging me too, I don't have a good solution for it